Makefile

```diff
@@ -306,11 +307,11 @@ remove_cert_manager:
 .PHONY: check_cert_manager
 check_cert_manager:
 	@api_versions=$$($(CLUSTER_CLIENT) api-versions) &&\
-       if [ $$(echo "$${api_versions}" | grep -c '^cert-manager.io/v1$$') -eq 0 ]; then if [ "$${DISABLE_SERVICE_TLS}" != "true" ]; then\
-                       echo 'cert-manager is not installed, install using "make cert_manager" or disable TLS for services by setting DISABLE_SERVICE_TLS to true' >&2\
-                       && exit 1;\
-               fi;\
-       fi
+		if [ $$(echo "$${api_versions}" | grep -c '^cert-manager.io/v1$$') -eq 0 ]; then if [ "$${DISABLE_SERVICE_TLS}" != "true" ]; then\
+						echo 'cert-manager is not installed, install using "make cert_manager" or disable TLS for services by setting DISABLE_SERVICE_TLS to true' >&2\
+					&& exit 1;\
+				fi;\
+		fi
```

Comment:
Thanks for cleaning up the other trailing spaces. This one seems to have gotten a little too much indentation. Converting to tabs is fine I think, but let's do one tab == one level of indentation.

Reply:
sorry about that was trying to follow the format that was used/dint want to change.